doc/manuals/admin/cli.md

```diff
@@ -86,7 +86,8 @@ The list of available options is the following:
 -   **-logAppend**. If used, the log lines are appended to the existing
     contextBroker log file, instead of starting with an empty log file.
 -   **-logLevel**. Select initial logging level, supported levels:
-    - NONE    (suppress ALL log output, including errors),
+    - NONE    (suppress ALL log output, including fatal error messages),
+    - FATAL   (show only fatal error messages),
```

Comment:
Changes in logs.md also needed. In particular:

* To add FATAL to the bullet list in the "Log file" section.
* Modify "lvl" entry, spliting current ERROR sub-bullet into ERROR + FATAL. In particular:
 * five -> six
 * "ERROR: This level designates error events. There is a severe problem that should be fixed. A subclass of ERROR is FATAL, which designates very severe error events that will presumably lead the application to abort. The process can no longer work" -> entry for FATAL: "designates very severe error events that will presumably lead the application to abort. The process can no longer work.", entry for ERROR: "This level designates error events. There is a severe problem that should be fixed."